### ISSUE TITLE
add 413 requested entity too large handling for bop

### ISSUE THREAD

**Issue by robnagler** [radiasoft/rsconf#24]

**Comment by robnagler** [comment]
@schellj you should decide if you want this done.

**Comment by schellj** [comment]
@robnagler is this for the ability to specify an error page for 413s? Would be nice to have as the user currently just gets the default nginx error page, though that probably doesn't happen much.

**Comment by robnagler** [comment]
Yes, we can write a custom page similar to maintenance down page. Just need to do it.
